doc/plugin_server_notifier_k8sbundle.md

```diff
@@ -14,6 +14,7 @@ The plugin accepts the following configuration options:
 | config_map            | The name of the ConfigMap                   | `spire-bundle`  |
 | config_map_key        | The key within the ConfigMap for the bundle | `bundle.crt`    |
 | kube_config_file_path | The path on disk to the kubeconfig containing configuration to enable interaction with the Kubernetes API server. If unset, it is assumed the notifier is in-cluster and in-cluster credentials will be used. | |
+| apiservice_label      | If set, rotate the CA Bundle in API services with this label set to `true`. | |
```

Comment:
How would you feel about `api_service_label`? `apiservice` lines up with kubectl but we've already departed from that with `config_map`...